plugins/redirect/gatsby-node.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable no-restricted-syntax */
+const path = require('path');
+const fs = require('node:fs/promises');
+
+const getMetaRedirect = require('./redirect');
+
+async function writeRedirectsFile(redirects, folder, pathPrefix) {
+  if (!redirects.length) return;
+
+  for (const redirect of redirects) {
```

Comment:
We shouldn't use `for` loops here as it will slow down the builds and require babel regenerator. Please use Promises.

Reply:
The only thing I changed was removing the fs-extra import and used the built-in fs module instead.

Reply:
> This is the same exact code as the plugin we were using. 🤷🏻‍♂️ it should work fine

Babel doesn't regenerate node modules by default if they're using CommonJS and we're using CommonJS. But it does in our source.